design/txn_author_agreement.md

```diff
@@ -83,27 +83,27 @@ Add new version of AML.
 Fetch AML from the ledger valid for specified time or the latest one.
 
 ### TAA Verification
-If TAA enabled on the ledger, then each write request from the user must contain metadata about a TAA signed by the user. The new format of write request is 
+If TAA enabled on the ledger, then each write request from the user must contain TAA acceptance signed by the user. The new format of write request is
 ```json=
-{ 
+{
     "reqId": INT,
     "operation" { ... },
     "identifier": "<str ident>",
     "protocolVersion": INT,
-    "signature": "<signature (txnAuthrAgrmtMeta is also signed)>",
-    "txnAuthrAgrmtMeta": { 
-        "acceptanceMechanismType": "label of type", 
-        "hash": "hash string", 
-        "timeOfAcceptance": <integer UTC TS>
+    "signature": "<signature (taaAcceptance is also signed)>",
+    "taaAcceptance": {
+        "hash": "hash string",
+        "mechanism": "label of type",
+        "time": <integer UTC TS>
```

Comment:
@andkononykhin @ashcherbakov @skhoroshavin  I'm in doubt is just `time` here unambiguous here or not. In general, in TAA flow "time" may mean
- time of signature
- time of acceptance
- time of TAA itself.

In this particular JSON the next level is "taaAcceptance" (not the original name) so I'm OK with this renaming, but not sure is it good idea or not

Reply:
I think that `time` here is unambigous - it is time of acceptance, since its under `taaAcceptance`. As for other variants:
- it cannot be time of signature or transaction since it is under very specific `taaAcceptance` section, and if we will need time of signature or transaction we definitely won't add them under this section - they just don't belong here
- it cannot be time of TAA itself because it is already contained in metadata of previously written TAA transaction